rfd/cspell.json

```diff
@@ -890,4 +891,4 @@
     "Zilla",
     "ZLMR"
   ]
-}
+}
```

Comment:
nit: missing newline, above too. ([rationale](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline))